scripts/windows-bundler.sh

```diff
@@ -76,11 +76,25 @@ cp "resources/Tachidesk Electron Launcher.bat" $release_name
 zip_name=$release_name.zip
 zip -9 -r $zip_name $release_name
 
+# create msi package
+sudo apt-get install -y wixl
+msi_name=$release_name.msi
+
+find $release_name/jre | wixl-heat --var var.SourceDir -p $release_name/ --directory-ref jre --component-group jre > jre.wxs
+find $release_name/electron | wixl-heat --var var.SourceDir -p $release_name/ --directory-ref electron --component-group electron > electron.wxs
+if [ $arch = "win32" ]; then
+  wixl -D SourceDir=$release_name --arch x86 Tachidesk-Server-x86.wxs jre.wxs electron.wxs -o $msi_name
+else
+  wixl -D SourceDir=$release_name --arch x64 Tachidesk-Server-x64.wxs jre.wxs electron.wxs -o $msi_name
+fi
+
 rm -rf $release_name
 
 # clean up from possible previous runs
 if [ -f ../server/build/$zip_name ]; then
   rm ../server/build/$zip_name
+if [ -f ../server/build/$msi_name ]; then
+  rm ../server/build/$msi_name
 fi
 
-mv $zip_name ../server/build/
+mv $zip_name $msi_name ../server/build/
```

Comment:
this doesn't seem to be correct

Reply:
Could you explain more. Shouldn't MSI file be moved to `server/build` ?

Reply:
mv takes two arguments, you are giving it 3

Reply:
But I can do this on my machine: `mv file1.ext1 file2.ext2 file3.ext3 /destination/`
Is it different on github's ubuntu?

Reply:
I'll change it. It doesn't make any difference anyway